decorator.rb

```diff
@@ -0,0 +1,26 @@
+require './nameable'
+
+class Decorator < Nameable
+  def initialize(nameable)
+    super()
+    @nameable = nameable
+  end
+
+  def correct_name
+    @nameable.correct_name
+  end
+end
+
+class CapitalizeDecorator < Decorator
+  def correct_name
+    # @nameable.correct_name.capitalize
+    super().capitalize
+  end
+end
+
+class TrimmerDecorator < Decorator
+  def correct_name
+    # @nameable.correct_name.strip
+    super().strip
+  end
```

Comment:
- [ ] The name has a maximum of 10 characters. If it's longer it should trim the word.
- [ ] Please remove commented code.
![Capture](https://user-images.githubusercontent.com/63972670/208798583-dd9169ce-28b5-483f-bdb0-3979d72ba6cb.JPG)